packages/spaces/commands/vpn/info.js

```diff
@@ -88,10 +94,11 @@ module.exports = {
   hidden: true,
   needsApp: false,
   needsAuth: true,
-  args: [{name: 'space', optional: true, hidden: true}],
+  args: [{name: 'space', optional: false, hidden: true}],
```

Comment:
For these args, I believe these refer to what's being passed into the command without a flag. As such, I think we need to be adding one here for `name` and removing the `space` parameter (so we require people use the flag to specify the space now).

Reply:
@watsonian yeah, I actually had them originally and agree.